presto-main/src/main/java/com/facebook/presto/operator/HashBuilderOperator.java

```diff
@@ -21,15 +21,18 @@
 import com.facebook.presto.sql.planner.Symbol;
```

Comment:
Please merge this with the `Calculate checksum of spilled LookupSource` so the single commit has both new code and usages